app-ios/Sources/AboutFeature/SafariView.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+import SafariServices
+
+public struct SafariView: UIViewControllerRepresentable {
+
+    private let url: URL
+    private let config: SFSafariViewController.Configuration
+
+    public init(url: URL, config: SFSafariViewController.Configuration = .init()) {
+        self.url = url
+        self.config = config
+    }
+
+    public func makeUIViewController(context: Context) -> SFSafariViewController {
+        let safariViewController = SFSafariViewController(url: url, configuration: config)
+        return safariViewController
+    }
+
+    public func updateUIViewController(_ uiViewController: SFSafariViewController, context: Context) {}
+}
```

Comment:
Can you create a new target about this?
I think this view is useful in any other views!

Example: 
- Create `SafariView` target
- Add `SafariView` target to `AboutFeature` dependency and use!

Reply:
@ry-itto 
I fixed at https://github.com/DroidKaigi/conference-app-2022/pull/451/commits/133b4a36d0a76ec3360019ef9686fcddffa9f03f !